crates/rattler_conda_types/src/channel/mod.rs

```diff
@@ -38,7 +38,7 @@ impl Default for ChannelConfig {
 }
 
 /// `Channel`s are the primary source of package information.
-#[derive(Debug, Clone, Serialize, Eq, PartialEq, Hash)]
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq, Hash)]
```

Comment:
This is no longer required right? Since you have a custom deserializer?

Reply:
Nup

Reply:
Removed